src/sentry/eventstream/kafka.py

```diff
@@ -18,30 +18,43 @@
 
 # Beware! Changing this, or the message format/fields themselves requires
 # consideration of all downstream consumers.
+EVENT_PROTOCOL_VERSION = 2
```

Comment:
Very open to feedback here (and see below).

We had previously added a few new actions to the existing `Protocol 1`. My thought there was net new (superset) functionality wouldn't/shouldn't hurt downstream readers. (I guess we never formally decided this was OK, but it was reviewed, so we might want to.)

Since the existing actions need to change, it feels like a lot to ask downstream to inspect the internal keys of messages to decide on what to do. Hence the bump to `Protocol 2` with the new changed keys for some actions.

It also makes a clean break 1 in that *all* consumers should be able to ignore non-`insert` messages from `Protocol 1`.

But as I said, really open to ideas/feedback/complaints here.

Reply:
Oh, and if we change this I need to identify/verify all downstream consumers. Is it just `snuba` and the post-processor? I think the post-processor can just look at Protocol `(0, 1, 2)` and type `insert` (for now) right?